Fixed azure ad related docs for idp groups import setup (#2268)
astronomer/docs: software/integrate-auth-system.md

  
    - **Name**: Any
    - **Supported account types**: Accounts in this organizational directory only (Astronomer only - single tenant)
    - **Redirect URIs**:
        - Web / `https://houston.BASEDOMAIN/v1/oauth/redirect/`.
        - Web / `https://houston.BASEDOMAIN/v1/oauth/callback/`.

>     Replace `BASEDOMAIN` with your own. For example, if your base domain is `mycompany.com`, your redirect URIs should be https://houston.mycompany.com/v1/oauth/redirect/ and https://houston.mycompany.com/v1/oauth/callback/.

Reviewer: ```suggestion
    Replace `BASEDOMAIN` with your own. For example, if your base domain is `mycompany.com`, your redirect URIs should be `https://houston.mycompany.com/v1/oauth/redirect/` and `https://houston.mycompany.com/v1/oauth/callback/`.
```

>     Replace `BASEDOMAIN` with your own. For example, if your base domain is `mycompany.com`, your redirect URIs should be https://houston.mycompany.com/v1/oauth/redirect/ and https://houston.mycompany.com/v1/oauth/callback/.

Reviewer: ```suggestion
    Replace `BASEDOMAIN` with your own. For example, if your base domain is `mycompany.com`, your redirect URIs should be `https://houston.mycompany.com/v1/oauth/redirect/` and `https://houston.mycompany.com/v1/oauth/callback/`.
```

4. Click **Add**.
5. Copy the values in the **Value** and **Secret ID** columns. 
6. Click **API permissions** in the left menu.
7. Click **Microsoft Graph** and add the following minimum permissions for Microsoft Graph:

    - `email`
    - `Group.Read.All`
    - `openid`
    - `profile`
    - `User.Read

>     `<br /> **Note:** Make sure that you have granted the Admin Consent for all these permissions.
>     ![azure_api_permissions_consent.png](/img/software/azure_api_permissions_consent.png)

Reviewer: ```suggestion
    
    For each of these permissions, select **Grant Admin Consent for Astronomer Data**. Your Microsoft Graph permissions should look similar to the following image:
    
    ![Completed permissions page in Azure](/img/software/azure_api_permissions_consent.png)
```

4. Click **Add**.
5. Copy the values in the **Value** and **Secret ID** columns. 
6. Click **API permissions** in the left menu.
7. Click **Microsoft Graph** and add the following minimum permissions for Microsoft Graph:

    - `email`
    - `Group.Read.All`
    - `openid`
    - `profile`

>     - `User.Read

Reviewer: ```suggestion
    - `User.Read`
```
